tests/functional/services/flows/test_flow_crud.py

```diff
@@ -23,12 +21,34 @@ def test_create_flow(flows_client, subscription_id):
         assert "subscription_id" not in req_body
 
 
+@pytest.mark.parametrize("value", [None, [], "dummy_value"])
+@pytest.mark.parametrize("key", ["run_managers", "run_monitors"])
+def test_create_flow_request_data(flows_client, key, value):
+    meta = load_response(flows_client.create_flow).metadata
+
+    # Remove the key or set as empty list
+    if value is None:
+        meta["params"].pop(key)
+    else:
+        meta["params"][key] = value
+
+    flows_client.create_flow(**meta["params"])
+
+    last_req = get_last_request()
+    req_body = json.loads(last_req.body)
+
+    if value is None:
+        assert key not in req_body
+    else:
+        assert req_body[key] == value
+
+
 def test_create_flow_error_parsing(flows_client):
-    metadata = load_response(
+    meta = load_response(
         flows_client.create_flow, case="bad_admin_principal_error"
     ).metadata
     with pytest.raises(FlowsAPIError) as excinfo:
-        flows_client.create_flow(**metadata["params"])
+        flows_client.create_flow(**meta["params"])
```

Comment:
As above, since we aren't changing this test, let's not modify variable names in it, to improve the usability of our git history.